cli/command/trust/sign.go

```diff
@@ -18,19 +18,26 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type signOptions struct {
+	local bool
+}
+
 func newSignCommand(dockerCli command.Cli) *cobra.Command {
+	options := signOptions{}
 	cmd := &cobra.Command{
 		Use:   "sign IMAGE:TAG",
 		Short: "Sign an image",
 		Args:  cli.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runSignImage(dockerCli, args[0])
+			return runSignImage(dockerCli, args[0], options)
```

Comment:
Minor: now that we have an options struct, the `imageName` can be passed as part of that struct, instead of a separate parameter. Not super important at this point since there is only a single arg, but it would be more consistent.